ln-resource-mgr/src/lib.rs

```diff
@@ -260,27 +264,35 @@ impl AllocationCheck {
     }
 }
 
-/// A snapshot of a reputation check for a htlc forward.
 #[derive(Clone, Debug, Eq, PartialEq, Serialize)]
 pub struct ReputationCheck {
-    pub outgoing_reputation: i64,
+    /// Reputation values of the incoming channel for the proposed htlc.
+    pub incoming_reputation: ReputationValues,
+    /// Reputation values of the outgoing channel for the proposed htlc.
+    pub outgoing_reputation: ReputationValues,
```

Comment:
nit: the values in `incoming_reputation` actually look at the incoming channel's reputation and the outgoing channel's revenue, so "of the incoming channel" doesn't really parse here (same for outgoing).

Perhaps: The values used to check incoming/outgoing reputation for the channel pair.